README.md

```diff
@@ -52,6 +51,7 @@ addon | version | summary
 [partner_auto_salesman](partner_auto_salesman/) | 8.0.1.0.0 (unported) | Partner auto salesman
 [partner_contact_address_detailed](partner_contact_address_detailed/) | 8.0.1.0.0 (unported) | All address data in summarized contact form
 [partner_contact_nationality](partner_contact_nationality/) | 9.0.1.0.0 (unported) | Add nationality field to contacts
+[partner_relations](partner_relations/) | 8.0.1.1.0 (unported) | Partner relations
```

Comment:
These changes to README.md should be removed. Module really moved to the migrated/ported list.